app/src/main/java/com/eventyay/organizer/ui/views/DatePicker.java

```diff
@@ -29,10 +37,23 @@ public void setValue(String value) {
             observableValue.set(value);
             String format = DateUtils.FORMAT_DATE_COMPLETE;
 
+            DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss", Locale.getDefault());
+            Date date = null;
+            try {
+                date = dateFormat.parse(value);
+            } catch (ParseException e) {
```

Comment:
I don't like this. Why?

Reply:
to convert a string into date object.. should i use substring ??

Reply:
Why catch?

Reply:
It was throwing an error that while parsing value, it must be enclosed in a try-catch block

Reply:
@iamareebjamal sir, it was throwing an error that parsing must be surrounded by try-catch block... Please review it as this error is making difficult to select date-time simultaneouly